### PR TITLE
Add: zenodo file to repo for citation

### PR DESCRIPTION
**Pull request by lwasser**

### CODE REVIEW

**Comment by lwasser** [comment]
@sneakers-the-rat @tupui   can i please add you as well to our zenodo file as a contributor here? You can add to this pr via a comment commit if you'd like, or submit separately or just provide your info here and i'll add you. many thanks.

**Comment by lwasser** [comment]
sorry y'all fixing a mess i made here rebasing :)

**Comment by lwasser** [comment]
fixed thanks to @xmnlab !! thanks y'all. will leave this open to try to get more peeps who contributed on the zenodo file!!

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        "name": "Molinsky, Simon",
+        "orcid": ""
+      },
+    ],
+    "contributors": [
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Marmo, Chiara",
+          "orcid": ""
```
Comment (cmarmo) [inline .zenodo.json:34]:
```suggestion
          "orcid": "0000-0003-2843-6044 "
```
Thanks! :pray:

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+          "type":"Other",
+          "affiliation": "",
+          "name": "Cano Rodríguez, Juan Luis",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Schreiner, Henry",
+          "orcid": ""
```
Comment (henryiii) [inline .zenodo.json:76]:
```suggestion
          "affiliation": "Princeton University",
          "name": "Schreiner, Henry",
          "orcid": "0000-0002-7833-783X"
```

File: .zenodo.json [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Schwartz, Eli",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
```
Comment (rgommers) [inline .zenodo.json:92]: Quansight

File: .zenodo.json [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+          "name": "Gommers, Ralf",
+          "orcid": ""
```
Comment (rgommers) [inline .zenodo.json:94]: 0000-0002-0300-3333

File: .zenodo.json [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+          "type":"Other",
+          "affiliation": "",
+          "name": "Ming, Frost",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "van Kemenade, Hugo",
+          "orcid": ""
```
Comment (hugovk) [inline .zenodo.json:124]: And mine is https://orcid.org/0000-0001-5715-8632:
```suggestion
          "orcid": "0000-0001-5715-8632"
```

File: .zenodo.json [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+          "type":"Other",
+          "affiliation": "",
+          "name": "Döring, Randy",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Cano Rodríguez, Juan Luis",
+          "orcid": ""
```
Comment (astrojuanlu) [inline .zenodo.json:70]:
```suggestion
          "orcid": "0000-0002-2187-161X"
```

✨

File: .zenodo.json [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
+          "name": "Schwartz, Eli",
+          "orcid": ""
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
```
Comment (lwasser) [inline .zenodo.json:92]:
```suggestion
          "affiliation": "Quansight",
```

**Comment by lwasser** [comment]
pre-commit.ci autofix

File: .zenodo.json [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+        },
+        {
+          "type":"Other",
+          "affiliation": "Quansight",
+          "name": "Pamphile T., Roy",
+          "orcid": "0000-0001-9816-1416"
+        },
+        {
+          "type":"Other",
+          "affiliation": "",
```
Comment (eriknw) [inline .zenodo.json:44]:
```suggestion
          "affiliation": "NVIDIA",
```

File: .zenodo.json [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+          "name": "Welch, Erik",
+          "orcid": ""
```
Comment (eriknw) [inline .zenodo.json:46]:
```suggestion
          "orcid": "0000-0003-3694-3783"
```

File: .zenodo.json (hunk 1 above)
Comment (lwasser) [reply .zenodo.json:82]: thank you!!

File: .zenodo.json (hunk 4 above)
Comment (lwasser) [inline .zenodo.json]:
```suggestion
          "name": "Gommers, Ralf",
          "orcid": "0000-0002-0300-3333"
```

**Comment by lwasser** [comment]
Thank you all! merging and releasing today!

**Pull request merged**